src/configure/configure.ts

```diff
@@ -165,15 +200,25 @@ class PipelineConfigurer {
 
     private async getAzureDevOpsDetails(): Promise<void> {
         if (!this.inputs.organizationName) {
-            let selectedOrganization = await extensionVariables.ui.showQuickPick(
-                this.azureDevOpsClient.listOrganizations().then((orgs) => orgs.map(x => { return { label: x.accountName }; })),
-                { placeHolder: Messages.selectOrganization });
-            this.inputs.organizationName = selectedOrganization.label;
+            this.inputs.isNewOrganization = false;
+            let devOpsOrganizations = await this.azureDevOpsClient.listOrganizations();
+            if(devOpsOrganizations && devOpsOrganizations.length > 0) {
+                let selectedOrganization = await extensionVariables.ui.showQuickPick(
+                    devOpsOrganizations.map(x => { return { label: x.accountName }; }), { placeHolder: Messages.selectOrganization });
+                this.inputs.organizationName = selectedOrganization.label;
+            }
+            else {
+                this.inputs.isNewOrganization = true;
+                this.inputs.organizationName = await extensionVariables.ui.showInputBox({
+                    placeHolder: Messages.enterAzureDevOpsOrganizationName, 
+                    validateInput: (organizationName) => this.azureDevOpsClient.validateOrganizationName(organizationName)
```

Comment:
Are we not auto deriving the account name as well ? If PM is on board then this is okay !

Reply:
Auto deriving organization name is separate user story being tracked in backlog, I will go ahead with this for now.